fineract-provider/src/main/java/org/apache/fineract/portfolio/account/jobs/executestandinginstructions/ExecuteStandingInstructionsTasklet.java

```diff
@@ -151,17 +160,21 @@ private boolean transferAmount(final List<Throwable> errors, final AccountTransf
             errorLog.append("Exception while trasfering funds ").append(e.getMessage());
 
         }
-        updateQuery.append(instructionId).append(",");
-        if (errorLog.length() > 0) {
+
+        if (!errorLog.isEmpty()) {
             transferCompleted = false;
-            updateQuery.append("'failed'").append(",");
-        } else {
-            updateQuery.append("'success'").append(",");
         }
-        updateQuery.append(accountTransferDTO.getTransactionAmount().doubleValue());
-        updateQuery.append(", now(),");
-        updateQuery.append("'").append(errorLog).append("')");
-        jdbcTemplate.update(updateQuery.toString());
+
+        final AccountTransferStandingInstructionsHistory newHistory = new AccountTransferStandingInstructionsHistory();
```

Comment:
I would move this whole logic into a repository

Reply:
Done